server/models/pepTalkModel.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+const pepTalkSchema = new mongoose.Schema({
+  talk: {
```

Comment:
Super minor documentation detail but I think our [database structure](https://www.notion.so/lablueprint/Database-Structure-15f05d3606c54a4d80d38da1481e3106?pvs=4) currently has this field as `prompt` instead of `talk`. It helps to update the database structure doc just so our Notion & MongoDB are consistent!